docs/source/basics/file_operators.md

```diff
@@ -0,0 +1,48 @@
+# Operators
+
+The Latch SDK offers several operators for grouping and organizing data, in a similar flavor to NextFlow.
+
+---
+
+## Joins
```

Comment:
Can we provide a basic example for each of the join? Just so people know what the expected inputs and outputs should look like.